src/component/admin/users/UsersList/UserListItem/UserListItem.styles.ts

```diff
@@ -16,13 +16,21 @@ export const useStyles = makeStyles()(theme => ({
             backgroundColor: unleashGrey[200],
             fontWeight: 'normal',
             border: 0,
-            '&:first-child': {
+            '&:first-of-type': {
                 borderTopLeftRadius: '8px',
                 borderBottomLeftRadius: '8px',
+                '&>button': {
+                    borderTopLeftRadius: '8px',
+                    borderBottomLeftRadius: '8px',
+                },
             },
```

Comment:
Something like this would avoid repetition:

```
'&:first-of-type': {
    '&, & > button': {
        borderTopLeftRadius: '8px',
        borderBottomLeftRadius: '8px',
    },
},
```